frontend/src/components/Home.js

```diff
@@ -10,11 +10,12 @@ import './Home.css'
 
 const getPosition = ( source, positions ) => positions[ source ] || { line: 0, page: 1 }
 
-const Home = ( { err, sources, positions } ) => (
+const Home = ( { err, sources, positions, version: { version } } ) => (
```

Comment:
Why is version an object containing version? It'll be better to simply pass in the string. Perhaps the server is returning an object, so it's best to resolve it there.